ocaml/xapi/extauth_plugin_ADpbis.ml

```diff
@@ -20,6 +20,55 @@ open D
 
 open Stdext.Xstringext
 
+let lwsmd_service = "lwsmd"
+
+module Lwsmd = struct
+
+  let is_ad_enabled ~__context =
+    Helpers.get_localhost ~__context
+    |> (fun self -> Db.Host.get_external_auth_type ~__context ~self)
+    |> (fun x -> x = Xapi_globs.auth_type_AD_Likewise)
+
+  let is_bootup() =
+    Fe_systemctl.is_active lwsmd_service
+
+  let is_stoped() =
+    not (is_bootup())
```

Comment:
I don't see the value of this function, just use `Fun.negate is_active` where it's needed.

Reply:
updated